src/handleAction.js

```diff
@@ -7,19 +7,15 @@ export default function handleAction(type, reducers, defaultState) {
     ? type.toString()
     : type;
 
+  const [nextReducer, throwReducer] = isFunction(reducers)
+    ? [reducers, reducers]
+    : [reducers.next, reducers.throw];
+
   return (state = defaultState, action) => {
     // If action type does not match, return previous state
     if (action.type !== typeValue) return state;
 
-    const handlerKey = action.error === true ? 'throw' : 'next';
-
-    // If function is passed instead of map, use as reducer
-    if (isFunction(reducers)) {
-      reducers.next = reducers.throw = reducers;
-    }
-
-    // Otherwise, assume an action map was passed
-    const reducer = reducers[handlerKey];
+    const reducer = action.error === true ? throwReducer : nextReducer;
 
     return isFunction(reducer)
       ? reducer(state, action)
```

Comment:
Yeah, it would seem that it's explicitly trying to not take dependencies. There's also https://github.com/acdlite/redux-actions/blob/master/src/createAction.js#L1 which is included in `_`. 

IMO we could quite easily factor out `isFunction` though, especially since it's already in the codebase. I think this would be more preferable than taking a dependency.

I've learned that forcing library users to take dependencies can be off-putting. So I've explicitly removed them where I could from my own libraries (in my case people were complaining about `_` and bundle size).